README.md

```diff
@@ -64,6 +67,25 @@ bind-key -n C-l if-shell "$is_vim" "send-keys C-l"  "select-pane -R"
 bind-key -n C-\ if-shell "$is_vim" "send-keys C-\\" "select-pane -l"
 ```
 
+#### Include the snippet
+
+The above code snippet is also available in the `vim-tmux-navigator.tmux`
+file, which can be includes like this in your tmux config:
+
+```tmux
+run-shell /path/to/vim-tmux-navigator/vim-tmux-navigator.tmux
+```
```

Comment:
I think this section is potentially confusing and I'd rather remove it altogether. While I appreciate the desire to make install easier, I've in general opted for clarity and directness rather than fully enumerating all the possibilities. With this section we're offering 3 different ways to set up. I'd prefer to remove this and clarify the other two to make this all clearer.